edwardpizarro-fatima.js

```diff
@@ -0,0 +1,222 @@
+//Question 1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+//console.log(calculateCube(5));
+//=> 125
+
+function calculateCube(){
+    console.log(5*25)
+  }
+   calculateCube()
+  //I wasnt sure about this solution, but it came out to 125. Im pretty sure there is a right way of doing this just not sure what it is.
```

Comment:
you are getting the right answer here, but you've hard-coded the function so that it only does the one calculation. 

you want to make this function so that it takes a number, like 5, and returns the cube of that number. The way we do this is with *parameters*. In the parentheses for the function, you want to add a placeholder that will represent the number that we will put in, like so:

`function calculateCube(num) {}`

After that, we need to add logic inside the curly brackets that does the math. cubing means multiplying a number by itself three times, so you can `return num*num*num`. The final result will look like:

```js
function calculateCube(num) {
    return 5*5*5
}
```

Parameters are SUPER DUPER important in all coding languages, so please reach out if you have any questions about them.